osf/models/node.py

```diff
@@ -778,15 +779,17 @@ def can_edit(self, auth=None, user=None):
 
     def get_aggregate_logs_query(self, auth):
         return (
-            (   # TODO: remove `list` call during phase 2
-                Q('node_id', 'in', list(Node.objects.get_children(self).can_view(user=auth.user, private_link=auth.private_link).values_list('id', flat=True))) |
-                Q('node_id', 'eq', self.id)
-            ) & Q('should_hide', 'eq', False)
+            (
+                Q(node_id__in=list(Node.objects.get_children(self).can_view(user=auth.user, private_link=auth.private_link).values_list('id', flat=True))) |
```

Comment:
`values_list` returns a list so you shouldn't need to call list on it, right? Or am I misreading the parenthesis?